packages/@lwc/engine-core/src/framework/reporting.ts

```diff
@@ -43,7 +76,7 @@ export const reportingControl = {
      *
      * @param dispatcher - reporting control
      */
-    attachDispatcher(dispatcher: ReportingDispatcher): void {
+    attachDispatcher(dispatcher: ReportingDispatcher<ReportingEventId>): void {
```

Comment:
With the suggestion above you don't need to specify the generic type value anymore.

```suggestion
    attachDispatcher(dispatcher: ReportingDispatcher): void {
```

Reply:
Neat, thanks!